modules/exploits/osx/browser/safari_createthis.rb

```diff
@@ -0,0 +1,466 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ManualRanking
+
+  include Msf::Exploit::EXE
+  include Msf::Exploit::Remote::HttpServer
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'Safari WebKit exploit',
+      'Description'    => %q{
+      },
+      'License'        => MSF_LICENSE,
+      'Author'         => [ 'saelo' ],
+      'References'     => [
+          ['CVE', '2018-4233'],
+          ['CVE', '2018-4404'],
+          ['URL', 'https://github.com/saelo/cve-2018-4233'],
+          ['URL', 'https://github.com/saelo/pwn2own2018'],
+          ['URL', 'https://saelo.github.io/presentations/blackhat_us_18_attacking_client_side_jit_compilers.pdf'],
+        ],
+      'Arch'           => [ ARCH_PYTHON, ARCH_CMD ],
+      'Platform'       => 'osx',
+      'DefaultTarget'  => 0,
+      'DefaultOptions' => { 'PAYLOAD' => 'python/meterpreter/reverse_tcp' },
+      'Targets'        => [
+          [ 'Python payload',  { 'Arch' => ARCH_PYTHON, 'Platform' => [ 'python' ] } ],
+          [ 'Command payload', { 'Arch' => ARCH_CMD, 'Platform' => [ 'unix' ] } ],
+        ],
+      'DisclosureDate' => 'Mar 15 2018'))
```

Comment:
Is this accurate? CVE says June and git repo is August.

Reply:
Hmm maybe the CVE is more accurate. I got this date from https://www.thezdi.com/blog/2018/3/14/pwn2own-2018-results-from-day-one

Reply:
I believe the disclosure date is meant to reflect the date of public disclosure. I didn't find any other references (from a quick search), and there's not enough information in the ZDI article to reproduce the vulnerability, beyond the knowledge that a vulnerability exists.